pkgs/development/libraries/SDL/default.nix

```diff
@@ -25,13 +25,14 @@ stdenv.mkDerivation rec {
 
   # Since `libpulse*.la' contain `-lgdbm', PulseAudio must be propagated.
   propagatedNativeBuildInputs =
```

Comment:
OK, I tried to disable the things on cygwin instead of making them native 043a847.